### ISSUE TITLE
Toggle options for Advancement Screenshot

### ISSUE THREAD

**Issue by sharkaccino** [ricksouth/serilum-mc-mods#2191]
## **Information**
Minecraft version: 1.20.1
Modloader: Fabric

Mod name: Advancement Screenshot

## **Feature description**
TL;DR: Either one or both of the following:
- An option to (at least temporarily) disable the mod's functionality.
- An option to disable automatic screenshots specifically while the player is in creative mode.

I often like to play in creative mode worlds alongside survival mode worlds from time to time. I love having this mod to capture key moments in my survival world, but it becomes far less helpful in my creative mode worlds, since I'm often just grabbing items from the creative inventory and suddenly getting tons of advancements all at once. I'm worried this will end up filling my hard drive, and I don't want to be constantly having to go out of my way to delete a bunch of useless screenshots. Technically I could also just disable the mod itself entirely, but that'd mean having to quit and restart the game every time I switch between survival and creative :(

Thanks for considering :raised_hands:

**Comment by ricksouth** [comment]
Hey!

I appreciate the suggestion! While I'd love to add all features submitted on the issue-tracker, I've got to prioritize working on them due to having limited time. In order to see which features users would like to see most, I've created https://serilum.com/mods/requests, which shows a table with all feature request submitted.

Users are able to upvote requests by reacting to the first issue comment on **this GitHub page** with one of the 👍 😀 🎉 ❤️ 🚀 👀 emoji's. The request with the most unique reactions, will be shown at the top. You can of course add the reactions yourself too, but don't have to :). The author of the feature is already counted as +1.

I won't only focus on popular features in the upcoming years, but it does help with prioritizing. I'll probably work on a combination of popular and interesting/needed/fun submissions.

I'll close this issue with "not planned" as a way to separate an open feature request from an actual completed issue. This does not actually mean it's not planned! Incompatibilities and bug reports will still remain open.

When the feature is implemented, I'll again post a comment and close it as "completed".

Thank you for taking the time to submit the suggestion! ❤️
